api/src/main/resources/workbench.yaml

```diff
@@ -1344,6 +1344,34 @@ paths:
             schema:
               $ref: "#/definitions/DomainValuesResponse"
 
+# Data Sets ###############################################################################
+
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/dataSet/generateQuery:
+      # Ideally this would be a get, but we need to pass up a request body for
```

Comment:
Feel free to remove this comment – I think with the new name, a POST feels more natural.